docs/manuals/core/config/scenario-config.md

```diff
@@ -15,77 +17,116 @@ A scenario configuration is necessary to instantiate a [Scenario](../concepts/sc
 
-# From pipeline configs
+# From task configs
+
+Here is a simple example assuming the task configuration `multiply_task_cfg` has already been created:
+
+```python linenums="1"
+from taipy import Config
+
+scenario_cfg = Config.configure_scenario("multiply_scenario",
+                                         [multiply_task_cfg])
+```
 
-!!! warning "Deprecation"
+In this example, we create a scenario configuration `ScenarioConfig^` from a task configuration already defined.
+A `ScenarioConfig^` must always be provided with a list of `TaskConfig^`.
 
-    Note that `PipelineConfig^` has been deprecated in version 2.3 and will be combined with `ScenarioConfig^` in the following major version.
-    It is recommended to configure ScenarioConfig with `configure_scenario_from_tasks` instead.
+# Adding additional data nodes configurations
 
-Here is a simple example assuming the pipeline configuration `pipeline_cfg` has already been created:
+Here is a simple example assuming the task configuration `multiply_task_cfg` and the data node configuration
+`additional_dn_cfg` have already been created:
 
 ```python linenums="1"
 from taipy import Config
 
 scenario_cfg = Config.configure_scenario("multiply_scenario",
-                                         [pipeline_cfg])
+                                         [multiply_task_cfg],
+                                         [additional_dn_cfg])
 ```
 
-In this example, we create a scenario configuration `ScenarioConfig^` from a pipeline configuration already defined.
+In this example, we create a scenario configuration `ScenarioConfig^` from a task configuration and
+a data node configuration already defined. The provided data node configuration represents additional data.
+Additional data node configurations are optional when configuring a `ScenarioConfig^`.
 
 
-# From task configs
+# Adding sequence descriptions
+
+In the scenario configuration, we can describe the sequences to create with the scenario creations.
+Here is a simple example assuming the task configuration `add_task_cfg_1`, `add_task_cfg_2`,
+`multiply_task_cfg_1`, `multiply_task_cfg_2` have already been created:
+
+```python linenums="1"
+from taipy import Config
+
+scenario_cfg = Config.configure_scenario("multiply_scenario",
+                                         [add_task_cfg_1, add_task_cfg_2, multiply_task_cfg_1, multiply_task_cfg_2],
+                                         sequences={"add_sequence": [add_task_cfg_1, add_task_cfg_2,],
+                                                    "multiply_sequence": [multiply_task_cfg_1, multiply_task_cfg_2,],})
+```
 
-When the scenario configuration contains only one single pipeline configuration, we can also create the
-`ScenarioConfig^` from the task configurations directly.
+We can also add sequences to your scenario configuration after you have created your scenario configuration.
```

Comment:
"We can also add sequences to your" -> "You can also add sequences to your"
(or "We can also add sequences to our" and change the end of the sentence)

Reply:
will change them all to "we" and "our" to be consistent with other examples